src/nrc/conf/includes/base.py

```diff
@@ -472,45 +79,10 @@
 ##############################
 
 # Django-axes
-AXES_CACHE = "axes"  # refers to CACHES setting
+# TODO these settings are different from OAF
```

Comment:
In OAF failure limit is 10 and cooloff time 1. Do we want to adapt these stricter settings instead?

Reply:
To be honest I have no opinion here. I think OAF settings are fine, but either way is ok